content/en/docs/reference/tools/map-crictl-dockercli.md

```diff
@@ -4,70 +4,10 @@ content_type: reference
-
-This page provides a reference for mapping common commands for the `docker`
-command-line tool into the equivalent commands for `crictl`.
-
-## Mapping from docker CLI to crictl
-
-The exact versions for the mapping table are for `docker` CLI v1.40 and `crictl`
-v1.19.0. This list is not exhaustive. For example, it doesn't include
-experimental `docker` CLI commands.
-
-{{< note >}}
-The output format of `crictl` is similar to `docker` CLI, despite some missing
-columns for some CLI. Make sure to check output for the specific command if your
-command output is being parsed programmatically.
-{{< /note >}}
-
-### Retrieve debugging information
-
-{{< table caption="mapping from docker cli to crictl - retrieve debugging information" >}}
-docker cli | crictl | Description | Unsupported Features
--- | -- | -- | --
-`attach` | `attach` | Attach to a running container | `--detach-keys`, `--sig-proxy`
-`exec` | `exec` | Run a command in a running container | `--privileged`, `--user`, `--detach-keys`
-`images` | `images` | List images |  
-`info` | `info` | Display system-wide information |  
-`inspect` | `inspect`, `inspecti` | Return low-level information on a container, image or task |  
-`logs` | `logs` | Fetch the logs of a container | `--details`
-`ps` | `ps` | List containers |  
-`stats` | `stats` | Display a live stream of container(s) resource usage statistics | Column: NET/BLOCK I/O, PIDs
-`version` | `version` | Show the runtime (Docker, ContainerD, or others) version information |  
-{{< /table >}}
-
-### Perform Changes
-
-{{< table caption="mapping from docker cli to crictl - perform changes" >}}
-docker cli | crictl | Description | Unsupported Features
--- | -- | -- | --
-`create` | `create` | Create a new container |  
-`kill` | `stop` (timeout = 0) | Kill one or more running container | `--signal`
-`pull` | `pull` | Pull an image or a repository from a registry | `--all-tags`, `--disable-content-trust`
-`rm` | `rm` | Remove one or more containers |  
-`rmi` | `rmi` | Remove one or more images |  
-`run` | `run` | Run a command in a new container |  
-`start` | `start` | Start one or more stopped containers | `--detach-keys`
-`stop` | `stop` | Stop one or more running containers |  
-`update` | `update` | Update configuration of one or more containers | `--restart`, `--blkio-weight` and some other resource limit not supported by CRI.
-{{< /table >}}
-
-### Supported only in crictl
-
-{{< table caption="mapping from docker cli to crictl - supported only in crictl" >}}
-crictl | Description
--- | --
-`imagefsinfo` | Return image filesystem info
-`inspectp` | Display the status of one or more pods
-`port-forward` | Forward local port to a pod
-`pods` | List pods
-`runp` | Run a new pod
-`rmp` | Remove one or more pods
-`stopp` | Stop one or more running pods
-{{< /table >}}
+{{<note>}}
+This page is being directed to 
+https://v1-24.docs.kubernetes.io/docs/reference/tools/map-crictl-dockercli/ because of
```

Comment:
Hmm; we're likely to shut down 
https://v1-24.docs.kubernetes.io/docs/reference/tools/map-crictl-dockercli/ soon.

See https://github.com/kubernetes/website/issues/45134 for a related discussion / request.

Reply:
ok thanks @sftim , What should we do with this PR. Should i remove reference of this page then rather than mentioning deprecation as suggested in https://github.com/kubernetes/website/pull/42017#issuecomment-1943175458

Reply:
Hi @sftim Is there any decision made for this PR. I will be happy to help in clarifying regarding this page.

Reply:
The SIG doesn't seem to have made a decision yet. I'll ask @kubernetes/sig-docs-pr-reviews on your behalf @Ritikaa96.

Reply:
Thanks @sftim

Reply:
The link is a 404 now. Please revise.

Reply:
Hi @tengqm , should i remove this page itself or you would rather suggest removing the link and rephrasing the proposed content. to simpler version

Reply:
I think we can safely remove this page.

Reply:
alright. I'm on it . thanks for confirmation

Reply:
Can we get an update on this PR? I agree the page should now be removed.

Reply:
Hi @stmcginnis , i'd been preoccupied with other work, i'll remove the page today.